jkube-kit/doc/src/main/asciidoc/inc/helm/_jkube_helm.adoc

```diff
@@ -167,6 +171,10 @@ Defaults to empty string.
   The parameters can also represent a Golang expression
 |
 
+| preserveParameterCase
+| A boolean toggling whether the parameter case is preserved when generating the Values.yaml file, off by default
```

Comment:
nit, would be better to write default value just like other fields.
```suggestion
| A boolean toggling whether the parameter case is preserved when generating the Values.yaml file

Defaults to `false`
```

Reply:
Done